File: app/src/main/java/com/example/thechallen_ge/Question.java
package com.example.thechallen_ge;

import java.util.Arrays;
import android.text.Html;

public class Question {
    String category;
    String type;
    String difficulty;
    String question;
    String correct_answer;
    String[] incorrect_answers;

    Question(String category, String type, String difficulty, String question, String correctAnswer,
             String[] incorrectAnswers) {
        this.category = category;
        this.type = type;
        this.difficulty = difficulty;
        this.question = question;
        this.correct_answer = correctAnswer;
        this.incorrect_answers = incorrectAnswers;

        for (int i = 0; i < incorrectAnswers.length; i++) {
            this.incorrect_answers[i] = Html.fromHtml(incorrectAnswers[i], Html.FROM_HTML_MODE_COMPACT).toString();
        }
    }

    public String toString() {
        String toReturn = "";

        toReturn += "Category: " + category;
        toReturn += "\n Type: " + type;
        toReturn += "\n Difficulty: " + difficulty;
        toReturn += "\n Question: " + question;
        toReturn += "\n CorrectAnswer: " + correct_answer;
        toReturn += "\n IncorrectAnswers: " + Arrays.toString(incorrect_answers);

        return toReturn;
    }
}

File: app/src/main/java/com/example/thechallen_ge/Question.kt
package com.example.thechallen_ge

import java.util.Arrays
import android.text.Html

class Question internal constructor(internal var category: String, internal var type: String, internal var difficulty: String, internal var question: String, internal var correct_answer: String,
                                    internal var incorrect_answers: Array<String>) {

    init {

        for (i in incorrect_answers.indices) {
            this.incorrect_answers[i] = Html.fromHtml(incorrect_answers[i], Html.FROM_HTML_MODE_COMPACT).toString()
        }
    }

    override fun toString(): String {
        var toReturn = ""

        toReturn += "Category: $category"
        toReturn += "\n Type: $type"
        toReturn += "\n Difficulty: $difficulty"
        toReturn += "\n Question: $question"
        toReturn += "\n CorrectAnswer: $correct_answer"
        toReturn += "\n IncorrectAnswers: " + Arrays.toString(incorrect_answers)

        return toReturn
    }
}